content/docs/user-guide/data-management/remote-storage/azure-blob-storage.md

```diff
@@ -219,3 +219,11 @@ See `dvc remote modify` for more command usage details.
 
 [Azure documentation]:
   https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/storage/azure-storage-blob#other-client--per-operation-configuration
+
+- `exclude_environment_credential` - bool. If `true`, excludes the environment credential source for Azure Remote.
```

Comment:
let's move it before the `Azure documentation` please (it should be a single list).